Assets/HoloToolkit/Input/Scripts/GazeManager.cs

```diff
@@ -2,135 +2,81 @@
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
 using UnityEngine;
+using UnityEngine.EventSystems;
 using UnityEngine.VR.WSA;
 
 namespace HoloToolkit.Unity
 {
     /// <summary>
-    /// GazeManager determines the location of the user's gaze, hit position and normals.
+    /// GazeManager makes the location of the user's gaze, hit position and normals accessible to other components.
     /// </summary>
     public partial class GazeManager : Singleton<GazeManager>
     {
-        [Tooltip("Maximum gaze distance, in meters, for calculating a hit.")]
-        public float MaxGazeDistance = 15.0f;
-
-        [Tooltip("Select the layers raycast should target.")]
-        public LayerMask RaycastLayerMask = Physics.DefaultRaycastLayers;
+        public GGVInputModule InputModule;
```

Comment:
It's part of a system that hasn't been fully integrated into this version of HoloToolkit, it seems. I'll revert the changes to GazeManager and stick with adding in the new Stabilization code separately.